custom_components/tuya_local/devices/eurom_sani_bathroom_towel_radiator.yaml

```diff
@@ -0,0 +1,152 @@
+name: Eurom Sani-Bathroom-Radiator Towel Wifi heater
+primary_entity:
```

Comment:
Ideally a products section should be here with the product id and branding as the name. The top level name should be generic in case other devices match.

Reply:
The I remove the Towel WiFi heater. "Eurom Sani Bathroom Radiator" is the common part in the name of the different models covered.

Reply:
The branding is the most important part to remove. Making it more general, such as "Bathroom radiator" is not an issue, but since it has the towel_temperature logic in there, it is probably only valid for the Towel warmer models.

Reply:
But like I said, finding the product id and keeping the branding info in a products section is optimum.

Reply:
The Product ID is some strange number, more like a serial than a recognizable product identifier. Maybe the product name makes more sense. But I don't know which part is specific for this model, since there is are black and white models, with different capacity values.
"model": "LBH-63",
"name": "Your Eurom Heater",
"product_id": "bc6g********jmjz",
"product_name": "LBH-63 TOWEL RAIL EUROM",

Reply:
By the way, for some reason, the integration no longer works since 1 day. Tuya cloud explorer still reports the same local key as I originally used. https://eu.iot.tuya.com/cloud/explorer

Reply:
> By the way, for some reason, the integration no longer works since 1 day. Tuya cloud explorer still reports the same local key as I originally used. https://eu.iot.tuya.com/cloud/explorer

Power-cycle the heater and it's working again. Strange that the Tuya app kept working, but no longer a HA - Tuya Local connection.